flink-ml-api/src/main/java/org/apache/flink/ml/api/core/Stage.java

```diff
@@ -39,6 +39,7 @@
     void save(String path) throws IOException;
 
     // NOTE: every Stage subclass should implement a static method with signature "static T
-    // load(String path)", where T refers to the concrete subclass. This static method should
-    // instantiate a new stage instance based on the data from the given path.
+    // load(StreamExecutionEnvironment env, String path)", where T refers to the concrete
```

Comment:
We might move this part of comment to the class comments? Then it would also appear in the java doc. 

Also change `static T load(StreamExecutionEnvironment env, String path)` -> `{@code static T load(StreamExecutionEnvironment env, String path)}` and `T` -> `{@code T}`

Reply:
Good point. The PR has been updated as suggested.